runtime/lua/vim/snippet.lua

```diff
@@ -222,6 +224,26 @@ end
 --- @field private _session? vim.snippet.Session
 local M = { session = nil }
 
+--- Displays the choices for the given tabstop as completion items.
+---
+--- @param tabstop vim.snippet.Tabstop
+local function display_choices(tabstop)
+  assert(tabstop.choices, 'Tabstop has no choices')
+
+  local start_col = tabstop:get_range()[2] + 1
+  local matches = vim.iter.map(function(choice)
+    return {
+      word = choice,
+      abbr = choice,
+      menu = '[Snippet]',
```

Comment:
I'm not sure about this menu value. I think that `[Snippet]` implies that this is a completion item with a snippet insert text, but "snippet choice" feels too long, and "choice" feels weird.

Reply:
I would just omit `abbr` (`word` is used when empty anyway) and `menu` here.